app/uk/gov/hmrc/cdsreimbursementclaimfrontend/controllers/rejectedgoodssingle/ServiceUnavailableController.scala

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.cdsreimbursementclaimfrontend.controllers.rejectedgoodssingle
+
+import com.google.inject.Inject
+import com.google.inject.Singleton
+import play.api.mvc.Action
+import play.api.mvc.AnyContent
+import uk.gov.hmrc.cdsreimbursementclaimfrontend.config.ViewConfig
+import uk.gov.hmrc.cdsreimbursementclaimfrontend.controllers.JourneyControllerComponents
+import uk.gov.hmrc.cdsreimbursementclaimfrontend.utils.Logging
+import uk.gov.hmrc.cdsreimbursementclaimfrontend.views.html.{rejectedgoods => pages}
+
+import scala.concurrent.ExecutionContext
+
+@Singleton
+class ServiceUnavailableController @Inject() (
+  val jcc: JourneyControllerComponents,
+  serviceUnavailablePage: pages.service_unavailable
+)(implicit val ec: ExecutionContext, viewConfig: ViewConfig)
+    extends RejectedGoodsSingleJourneyBaseController
+    with Logging {
+
+  val show: Action[AnyContent] = actionReadJourney { implicit request => _ =>
```

Comment:
I think we can use simple action here, no need to read journey

Reply:
When I use the simple simpleActionReadJourney, I get an error: Cannot find any HTTP Request here 
And in the code: No implicits found for parameters request: Request[_] messages: Messages

I used the actionReadJourney as there were no errors using it